app.rambox.Rambox.metainfo.xml

```diff
@@ -0,0 +1,33 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>app.rambox.Rambox</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <name>Rambox Community Edition</name>
+  <developer_name>Rambox</developer_name>
+  <url type="homepage">https://rambox.app/#ce</url>
+  <summary>Put all your online Services in one Application!</summary>
+  <url type="bugtracker">https://github.com/ramboxapp/community-edition/issues</url>
+  <description>
+    <p>Rambox gives you the possibility to add common services many times you need, all in one place. It is completely open source</p>
+    <p>This Program let you create separate containers for different services and accounts in tabs and checks for notifications in the Background. This basically lets you convert as many websites as you want into desktop Application without having to Install </p>
+		<p>It's perfect for people who work with many services for business and private accounts. (unofficial Flatpak)</p>
```

Comment:
Both summary and descriptions should be generally improved using information from project's website.